3p/embedly.js

```diff
@@ -98,15 +98,15 @@ export function embedly(global, data) {
 
   container.appendChild(card);
 
-  getEmbedly(global, function() {
+  getEmbedly(global, function () {
```

Comment:
Ick.

Reply:
☹️ Doesn't appear to be a way to override this: https://prettier.io/docs/en/options.html